### PR TITLE
feat: Add VPC, EC2 Python Example

### PR DESCRIPTION
**Pull request by huangzbaws**
This pull request is aimed at creating the following Python Examples:
- Import VPC
- Create new VPC with 2 AZs and 6 Subnets, Public, Private and DataBase.
- Sharing VPC between two stacks
- Create ALB/EC2/AutoscalingGroup in the VPC
- How to define EC2 to specify AMI or auto-selection
- Define property override for EC2 Instance with BlockStorageMapping
- Define userdata for EC2 to setup httpd
- Create NAT GW and Bastion 
- Chain the Security Groups
---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Comment by huangzbaws** [comment]
change PR title

**Comment by huangzbaws** [comment]
@Mergifyio refresh

File: python/existing-vpc-new-ec2-ebs-userdata/app.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+from aws_cdk import core
+
+from cdk_vpc_ec2.cdk_vpc_ec2_stack import CdkVpcEc2Stack
+
+env_cn = core.Environment(account="1111111111", region="cn-northwest-1")
```
Comment (NGL321) [inline python/existing-vpc-new-ec2-ebs-userdata/app.py]: Leave inline comment indicating this needs to be replaced with personal account details

File: python/existing-vpc-new-ec2-ebs-userdata/cdk_vpc_ec2/cdk_vpc_ec2_stack.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+from aws_cdk import core
+import aws_cdk.aws_ec2 as ec2
+
+vpc_id = "vpc-111111"  # Import an Exist VPC
+ec2_type = "m5.xlarge"
+key_name = "id_rsa"
+linux_ami = ec2.GenericLinuxImage({
+    "cn-northwest-1": "ami-0f62e91915e16cfc2",
+    "eu-west-1": "ami-1111111"
```
Comment (NGL321) [inline python/existing-vpc-new-ec2-ebs-userdata/cdk_vpc_ec2/cdk_vpc_ec2_stack.py]: Please indicate inline placeholder

File: python/existing-vpc-new-ec2-ebs-userdata/cdk_vpc_ec2/cdk_vpc_ec2_stack.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+from aws_cdk import core
+import aws_cdk.aws_ec2 as ec2
+
+vpc_id = "vpc-111111"  # Import an Exist VPC
+ec2_type = "m5.xlarge"
```
Comment (richardhboyd) [inline python/existing-vpc-new-ec2-ebs-userdata/cdk_vpc_ec2/cdk_vpc_ec2_stack.py]: Can this be a t2.micro? We try to bias towards free-tier eligible applications as much as possible for demo code.

**Comment by huangzbaws** [comment]
Add readme
Add inline placeholder of account id, AMI id 
Change default ec2 type to t2.micro
Add RDS stack to the project
@NGL321 @richardhboyd

File: python/existing-vpc-new-ec2-ebs-userdata/cdk_vpc_ec2/cdk_vpc_ec2_stack.py (hunk 3 above)
Comment (NGL321) [reply python/existing-vpc-new-ec2-ebs-userdata/cdk_vpc_ec2/cdk_vpc_ec2_stack.py]: Good catch

**Pull request merged**